options/locale/locale_en-US.ini

```diff
@@ -1138,17 +1138,17 @@ users.update_profile_success = Account profile has been updated.
 users.edit_account = Edit Account
 users.max_repo_creation = Maximum Repository Creation Limit
 users.max_repo_creation_desc = (Set -1 to use global default limit)
-users.is_activated = This account has completed activation
-users.prohibit_login = This account is blocked from logging in
-users.is_admin = This account has administrator permissions
-users.allow_git_hook = This account has permission to create Git hooks
-users.allow_import_local = This account has permissions to import local repositories
-users.allow_create_organization = This account has permissions to create organizations
+users.is_activated =  Account activated
+users.prohibit_login = Login disabled
+users.is_admin = Administrator permissions
+users.allow_git_hook = Allowed to create git hooks
+users.allow_import_local = Allowed to import local repositories
+users.allow_create_organization = Allowed to create organizations
```

Comment:
The above ones are checkboxes on the user edit page, so no need to say `This account`. This is why I changed these lines.